tests/semantic_analyser.cpp

```diff
@@ -2971,13 +2972,6 @@ TEST_F(SemanticAnalyserTest, enums)
   test("enum named { a = 1, b } kprobe:f { printf(\"%s\", a); }");
   test("enum named { a = 1, b } kprobe:f { $e = a; printf(\"%s\", $e); }");
   test("enum named { a = 1, b } kprobe:f { printf(\"%15s %-15s\", a, a); }");
-
-  // Cannot symbolize a non-enum
-  test("kprobe:f { $x = (uint8)1; printf(\"%s\", $x) }", Error{ R"(
```

Comment:
Should we replace this with a positive test?

Reply:
You can now `%s` any type, as it is merely the `TextOutput` representation of that primitive. I'm not inclined to add every one of those cases individually as tests? But open to suggestions.

Reply:
We don't have to be exhaustive but a few positive tests with some primitives that didn't previously work when trying to be printed as strings would be nice.

Reply:
Will do